<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewreservation</class>
 <widget class="QDialog" name="addnewreservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrival date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="arr_dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>18</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2021</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateEdit" name="dep_dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>18</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_txt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Name Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Room:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinPrice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2147483647</number>
   </property>
  </widget>
  <widget class="QPushButton" name="recalc_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Recalculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NIP:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nip_txt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone number:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone_txt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>123-456-789</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_reservation_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_but">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="roomLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
